src/components/BoxVaultAPY/BoxVaultAPY.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { MouseEvent, ReactElement } from "react";
+import Token from "@yearn/yearn-web-assets";
+import styled, { css } from "styled-components";
+import { Heading1, Text } from "../../typography";
+import { Icon } from "..";
+import { IconDirection } from "../Icon/Direction";
+
+export interface BoxVaultAPYProps {
+  address: string;
+  chainID?: "1" | "250" | "42161";
+  imgFormat?: "img" | "svg";
+  label: string;
+  value: string;
+  onClick?: (event: MouseEvent<HTMLButtonElement>) => void;
+}
+
+export const BoxVaultAPY = ({
+  address,
+  chainID,
+  label,
+  value,
+  imgFormat,
+  ...props
+}: BoxVaultAPYProps): ReactElement => (
+  <StyledBox {...props}>
+    <StyledCol>
+      <div style={{ marginRight: "1rem" }}>
+        <Token
+          format={imgFormat ?? "svg"}
+          address={address}
+          chainID={chainID ?? "1"}
+          width={64}
+          height={64}
+        />
+      </div>
+      <div>
+        <StyledText>{label}</StyledText>
+        <StyledHeading1 bold forwardedAs={"p"}>
+          {value}
+        </StyledHeading1>
+      </div>
+    </StyledCol>
+    <div>
+      <span onClick={() => null}>
```

Comment:
We should get pass the `onClick` prop here, right?

```suggestion
      <span onClick={props.onClick}>
```

Reply:
We could actually just use the new button icon here, if you want to merge https://github.com/Major-Eth/yearn-web-lib/pull/23 then we can use it here

Reply:
Yes and no, the whole box should be clickable, not only this part. I am removing the `<span>` !